authz/api/grpc/client.go

```diff
@@ -0,0 +1,67 @@
+package grpc
+
+import (
+	"context"
+	"time"
+
+	"github.com/go-kit/kit/endpoint"
+	kitgrpc "github.com/go-kit/kit/transport/grpc"
+	pb "github.com/mainflux/mainflux/authz/api/pb"
```

Comment:
Do you need `pb` custom name for a package here - it is already called `pb`, no?

Reply:
yes, right, but lets see if this proto is going to stay here or some other place suits more